LEGO1/lego/legoomni/include/legoanimationmanager.h

```diff
@@ -155,7 +155,9 @@ class LegoAnimationManager : public MxCore {
 	MxBool FindVehicle(const char* p_name, MxU32& p_index);
 	MxResult ReadAnimInfo(LegoFile* p_file, AnimInfo* p_info);
 	MxResult ReadModelInfo(LegoFile* p_file, ModelInfo* p_info);
+	void FUN_10060480(LegoChar* p_param1[], int p_param2);
```

Comment:
Shouldn't use `int` here (`MxS32` if you are sure it's that? otherwise `undefined4`)